classes/class.ilMumieTaskGradeOverviewGUI.php

```diff
@@ -180,21 +178,14 @@ public function getAlert()
         return "";
     }
 
-
-
-    public function getPostVar()
-    {
-        return $this->postvar;
-    }
-
     /**
     * Get Post Variable.
     *
     * @return	string	Post Variable
     */
     public function getFieldId()
     {
-        $id = str_replace("[", "__", $this->getPostVar());
+        $id = str_replace("[", "__", null);
```

Comment:
See my other comment about str_replace

Reply:
removed strreplace entirely and set return to null

Reply:
removed strreplace entirely and set return to null

Reply:
Do we really need a function to retrun null?

Reply:
no return value needed but the function itself is